### PR TITLE
Add zoom links to live page

### PR DESCRIPTION
**Pull request by corcra**
Adds the links to zoom tutorials for day 1 and day 2 to the live page. Does not include the passwords for said meetings.

I was fighting with the flexbox that the snowman and gather text were contained in, so I just removed it for simplicity. This section fo text now matches the rocket.chat instructions in aesthetics.

### CODE REVIEW

**Comment by kperros** [comment]
Hmm, while I like the simplicity/cleanliness of the current version, I am a bit worried if now the text below the "Chat" Section is not distinct enough from the rest of the webpage contents?

Is there any way in which we can create a different Section which seems separate to the "Chat" one, which we name sth like "Useful Links" and we place the Zoom / Gather / Schedule links there?

BTW, I am wondering if it might also make sense to rename the title "CHIL Conference 2022" to "Livestream"?

**Comment by corcra** [comment]
I've added a section at the bottom of the live page which looks like this: 
image

**Comment by briangow** [comment]
@corcra , looks good to me!

File: templates/content/live.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-<div class="gather-town">
-    <img class="gather-town-avatar" src="/static/images/gather-avatar.png" alt="gather town avatar"/>
-    <p class="gather-town-text">Join us in <a href="https://app.gather.town/app/j4TbOZN9zwDb9xsP/CHIL">GatherTown</a> for breakouts, sponsors, posters, and socialising.</p>
-    <!-- See <a href="/calendar.html#tab-schedule" rel="noopener">schedule</a> for more details. -->
-</div>
+<h6 align="left"><p>Go to <b>Zoom</b> for <b>Tutorials</b>. Day 1 link <a href="https://us06web.zoom.us/j/81140384540">here</a>. Day 2 link <a href="https://us06web.zoom.us/j/81712359352">here</a>.</p>
```
Comment (kperros) [inline templates/content/live.md:1]:
```suggestion
<h6 align="left"><p>Tutorials are hosted in Zoom. For Day 1, the link can be found <a href="https://us06web.zoom.us/j/81140384540">here</a>. For Day 2, the link can be found <a href="https://us06web.zoom.us/j/81712359352">here</a>.</p>
```

File: templates/content/live.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+<p>Go to <b><a href="https://app.gather.town/app/j4TbOZN9zwDb9xsP/CHIL">Gather</a></b> (<a href="https://app.gather.town/app/j4TbOZN9zwDb9xsP/CHIL">link</a>) for
```
Comment (kperros) [inline templates/content/live.md:2]:
```suggestion
<p>Gather (<a href="https://app.gather.town/app/j4TbOZN9zwDb9xsP/CHIL">link</a>) is used for:
```

**Review by kperros — COMMENTED** [review]
Tried to align the formatting across platforms to be the same (i.e., we didn't use any bold for rocket.chat); also removed the double link that existed for Gather; finally, changed the wording a bit to align with the email we are going to send out a day before the conf.

Let me know if those changes make sense, thanks!

**Comment by corcra** [comment]
Sounds good, thanks @kperros !

**Pull request merged**